examples/k8s-sidecar/proxy_with_sa_key.yaml

```diff
@@ -55,6 +55,12 @@ spec:
           # following flag to have the proxy connect over private IP
           # - "-ip_address_types=PRIVATE"
 
+          # By default, the proxy will write all logs to stderr. In some
+          # environments, anything printed to stderr is consider an error. To
+          # disable this behavior and write all logs to stdout (except errors
+          # which will still go to stderr), use:
+          # - "-log_debug_stdout"
```

Comment:
nit: I'd suggest just uncommenting this - it's probably the behavior we want to suggest